304-quarkus-vertx-routes/src/main/java/io/quarkus/qe/validation/annotations/UppercaseService.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.qe.validation.annotations;
+
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class UppercaseService {
```

Comment:
This class is not really necessary. You could move on these validations to `UppercaseValidator`

Reply:
I know it's not necessary, but I wanted to verify that we can inject services in our custom validators.

Reply:
aka intentional, resolving and will merge